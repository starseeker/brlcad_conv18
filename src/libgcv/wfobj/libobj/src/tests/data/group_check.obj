# check for correct object tagging of polygons

g EMPTY1 EMPTY1 # should be 1 empty group, tagging for elements only
v 0 0 0
v 0 1 0
v 1 1 0
v 1 0 0

g EMPTY2
vt 0 0 0
vt 0 1 0
vt 1 1 0
vt 1 0 0

vn 0 0 0
vn 0 1 0
vn 1 1 0
vn 1 0 0

g a b c
f 1 2 3
g a
f 1/1/1 3/3/3 4/4/4

v 0 0 1
v 0 1 1
v 1 1 1
v 1 0 1

vt 0 0 1
vt 0 1 1
vt 1 1 1
vt 1 0 1

vn 0 0 1
vn 0 1 1
vn 1 1 1
vn 1 0 1

g d e f
g h i j
f 5/5 6/6 7/7
g e
f 5//5 7//7 8//8
