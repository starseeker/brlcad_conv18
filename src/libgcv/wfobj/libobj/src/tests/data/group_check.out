vertices {
  <0,0,0,1>
  <0,1,0,1>
  <1,1,0,1>
  <1,0,0,1>
  <0,0,1,1>
  <0,1,1,1>
  <1,1,1,1>
  <1,0,1,1>
}
texture coordinates {
  <0,0,0>
  <0,1,0>
  <1,1,0>
  <1,0,0>
  <0,0,1>
  <0,1,1>
  <1,1,1>
  <1,0,1>
}
normals {
  <0,0,0>
  <0,1,0>
  <1,1,0>
  <1,0,0>
  <0,0,1>
  <0,1,1>
  <1,1,1>
  <1,0,1>
}
face {
  groups: a b c
  object: ''
  material: ''
  materiallibs: 
  texmap: ''
  texmaplibs: 
  shadow_obj: ''
  trace_obj: ''
  smoothing group: '0'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '0'
  vertices: 1 2 3 
}
textured face {
  groups: h i j
  object: ''
  material: ''
  materiallibs: 
  texmap: ''
  texmaplibs: 
  shadow_obj: ''
  trace_obj: ''
  smoothing group: '0'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '0'
  vertices: 5/5 6/6 7/7 
}
oriented face {
  groups: e
  object: ''
  material: ''
  materiallibs: 
  texmap: ''
  texmaplibs: 
  shadow_obj: ''
  trace_obj: ''
  smoothing group: '0'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '0'
  vertices: 5//5 7//7 8//8 
}
textured oriented face {
  groups: a
  object: ''
  material: ''
  materiallibs: 
  texmap: ''
  texmaplibs: 
  shadow_obj: ''
  trace_obj: ''
  smoothing group: '0'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '0'
  vertices: 1/1/1 3/3/3 4/4/4 
}
